.github/workflows/check-manifest.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: '3.8'
+          python-version: '3.9'
```

Comment:
Are you suggesting I remove this? I believe it might make sense to keep this explicit since this is a C extension that may need to be pinned to a different version than what is the default. If you'd like me to remove it I can.